cms/toolbar/utils.py

```diff
@@ -117,14 +120,41 @@ def get_toolbar_from_request(request):
     return getattr(request, 'toolbar', EmptyToolbar(request))
 
 
+def add_live_url_querystring_param(obj, url, language=None):
+    """
+    Get CMS_ENDPOINT_LIVE_URL_QUERYSTRING_PARAM as the parameter, and the live url, if the object is
+    PageContent
+    """
+    url_param = getattr(settings, 'CMS_ENDPOINT_LIVE_URL_QUERYSTRING_PARAM', None)
```

Comment:
```suggestion
    url_param = get_cms_setting('ENDPOINT_LIVE_URL_QUERYSTRING_PARAM')
```

from cms.utils.conf import get_cms_setting

This isn't how you get settings in the core.